openmc/lattice.py

```diff
@@ -378,6 +366,51 @@ def get_all_universes(self):
 
         return all_universes
 
+    def get_universe(self, idx):
+        """Return universe corresponding to a lattice element index
+
+        Parameters
+        ----------
+        idx : Iterable of int
```

Comment:
Yes, `get_universe_index` returns indices that are used in the `universes` property, so they are (z, y, x).